p4runtime_sh/shell.py

```diff
@@ -584,7 +584,7 @@ def _get_param(self, name):
         if name not in self._params:
             raise UserError(
                 "'{}' is not a valid action parameter name for action '{}'".format(
-                    name, self._action_name))
+                    name, self.action_name))
```

Comment:
This appeared to be a typo.  During my testing, I hit this `raise` call and it caused an exception when not being able to find an attribute with this name.